build/DeployTasks/DeployVsixToGitHubTask.cs

```diff
@@ -0,0 +1,70 @@
+﻿
+using System.Net.Http.Headers;
+using System.Text.Json;
+
+namespace BuildScripts;
+
+[TaskName("DeployVsixToGitHubTask")]
+[IsDependentOn(typeof(DownloadArtifactsTask))]
+public sealed class DeployVsixToGitHubTask : FrostingTask<BuildContext>
```

Comment:
Unless I'm mistaken, there is no purpose to this task as you are already uploading the .vsix file in the main .yaml file above.

Reply:
File removed.